azure-pipelines.yml

```diff
@@ -16,3 +16,9 @@ jobs:
       py37-x64:
         PYTHON_VERSION: '3.7'
         PYTHON_ARCH: 'x64'
+      py37-x64:
+        PYTHON_VERSION: '3.8'
+        PYTHON_ARCH: 'x64'
+      py37-x64:
```

Comment:
```suggestion
      py38-x64:
```

Reply:
Azure is not running anyway... Or is not shown by the merge button

Reply:
[This may be the reason](https://dev.azure.com/physiopy/phys2bids/_build/results?buildId=709&view=results).
That's where my suggestions come from!

Reply:
Oh, I see! Let's commit them.